corehq/motech/repeaters/tests/test_models.py

```diff
@@ -41,6 +44,48 @@ def test_get_all_repeater_types():
         assert_in(name, types)
 
 
+class TestSQLCaseRepeater(TestCase):
+    def setUp(self):
+        self.url = "http://example.com"
+        self.conn = ConnectionSettings.objects.create(domain=DOMAIN, name=self.url, url=self.url)
+        return super().setUp()
+
+    def tearDown(self):
+        delete_all_repeaters()
+        return super().tearDown()
+
+    def _assert_same_repeater_objects(self, sql_repeater, couch_repeater):
+        self.assertEqual(sql_repeater.domain, couch_repeater.domain)
+        self.assertEqual(sql_repeater.couch_id, couch_repeater._id)
+        self.assertEqual(sql_repeater.is_paused, couch_repeater.paused)
+        self.assertEqual(sql_repeater.white_listed_case_types, couch_repeater.white_listed_case_types)
+        self.assertEqual(sql_repeater.black_listed_users, couch_repeater.black_listed_users)
+        self.assertEqual(sql_repeater.connection_settings.id, couch_repeater.connection_settings_id)
+
+    def test_repeaters_are_synced_to_sql(self):
+        couch_repeater = CaseRepeater(
+            domain=DOMAIN,
+            connection_settings_id=self.conn.id,
+            format='case_json'
+        )
+        couch_repeater.save()
+        sql_repeater = SQLCaseRepeater.objects.first()
```

Comment:
This assumes there is only one `SQLCaseRepeater` row in the database, which may not be true if some other test did not clean up properly (and that is generally very hard to track down).
```suggestion
        sql_repeater = SQLCaseRepeater.objects.get(couch_id=couch_repeater._id)
```